src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -128,6 +140,38 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
         filteredPersons.setPredicate(predicate);
     }
 
+    //=========== Filtered Record List Accessors =============================================================
+
+    /**
+     * Returns an unmodifiable view of the list of {@code Record} backed by the internal list of
+     * {@code versionedAddressBook}
+     */
+    @Override
+    public ObservableList<Record> getFilteredRecordList() {
```

Comment:
Regarding this method, could it possibly be implemented in the RecordList class? Similarly for the updateFilteredRecordList() method below.

Reply:
I don't think it should since modelmanager is handling the filtered list to be displayed while recordlist is handling the internal implementation of the list.